kolibri/utils/cli.py

```diff
@@ -1,17 +1,25 @@
+"""
+We do not import unicode_literals here as it causes Click to
+give some very aggressive warning errors about subtle bugs.
+So, to avoid these errors and these subtle bugs, we use
+explicit unicode literals in the following.
+"""
 from __future__ import absolute_import
 from __future__ import print_function
-from __future__ import unicode_literals
```

Comment:
Sorry, first I commented on this without reading the comment above :)

I've now seen these two sources:

1. https://github.com/PythonCharmers/python-future/issues/22
1. https://click.palletsprojects.com/en/7.x/python3/#unicode-literals

I understand that the motivation from discouraging this in `python-future` and `Click` doesn't have anything to do with Click specifically.

Since we have this import throughout the project, IMO it's fine. Most of the historic issues of why Python 2.7 had unicode issues have been fixed, and Python 2.7 support in itself isn't really encouraged.

I think it's okay to re-introduce this, considering that everything else in the project wants unicode for Python 2.7.

If you agree, I can open a new issue. CC: @rtibbles

Reply:
They advice not to do this:

```
import click
click.disable_unicode_literals_warning = True
```

...but that could be done if we feel comfortable about it. I don't quite understand if there are issues pertaining unicode and Python 2.7 for Click, or if they are just doing this as some general discouragement. From their own docs and the referred issue, I don't see any reasoning that isn't more general, and I think the consequence of agreeing to this, would be to refactor our whole codebase to be explicitly unicode `u"string"`.

Reply:
that reasoning makes sense to me - I'd personally be OK with setting `click.disable_unicode_literals_warning = True` for the sake of string consistency.

@benjaoming comments on PRs that are already merged are liable to get lost. If you feel that this something that should be addressed, please open a follow-up PR or issue

Reply:
@indirectlylit choosing the code comment because I needed clarity before opening an issue/PR

> If you agree, I can open a new issue. CC: @rtibbles

Will do that!

Thanks for weighing in both! :heart: